src/Symfony/Component/DependencyInjection/Compiler/RegisterAutoconfigureAttributesPass.php

```diff
@@ -49,36 +49,41 @@ public function accept(Definition $definition): bool
 
     public function processClass(ContainerBuilder $container, \ReflectionClass $class)
     {
+        $combinedAttributesArgs = null;
         foreach ($class->getAttributes(Autoconfigure::class, \ReflectionAttribute::IS_INSTANCEOF) as $attribute) {
-            self::registerForAutoconfiguration($container, $class, $attribute);
+            $args = array_filter((array) $attribute->newInstance(), static function ($v): bool { return null !== $v; });
+
+            foreach ($args['tags'] ?? [] as $i => $tag) {
+                if (\is_array($tag) && [0] === array_keys($tag)) {
+                    $args['tags'][$i] = [$class->name => $tag[0]];
+                }
+            }
```

Comment:
Note that with your PR, only the last tag is configured, whereas all the tags were previously added for this attribute:

```php
#[AutoconfigureTag('foo', ['priority' => 2])]
#[AutoconfigureTag('bar')]
```

Reply:
Alright, better now. I deliberately chose to only support merging tags and calls, because these are the only things in the `ChildDefinition` that have adder methods (`addTag()` and `addMethodCall()`). Other properties, like bindings, does not have an adder, but only `setBindings()`.

But I'm open to discussion on this point, and it would make sense to also merge bindings and properties?

Reply:
Seems correct. That matches the current behavior. We could add tests on these properties too.